Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -383,10 +382,9 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition& GetPosition,
     amrex::ParallelFor(
         np_to_depose,
         [=] AMREX_GPU_DEVICE (long const ip) {
-#if (defined AMREX_USE_GPU)
             KernelTimer KnlTimer(cost && load_balance_costs_update_algo
```

Comment:
Oh, I meant "let's replace this everywhere" ;-)
```suggestion
            KernelTimer kernelTimer(cost && load_balance_costs_update_algo
```